odinaka-joy/covid19-estimator-js
# completes challenge one

#### What does this PR do?
It completes challenge one of #BuildForSDG Cohort-1 JavaScript Assessment

#### Description of Task to be completed?
- It generates currentlyInfected value for both impact and severeImpact
- It generates infectionsByRequestedTime value for both impact and severeImpact

#### How should this be manually tested?
- Clone the repo and CD into project folder
- Run 'npm install' to install all projects dependencies
- Run 'npm run lint' to ensure project is error free
